_posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown

```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title: "Book Review - Super Pumped: The Battle for Uber, Mike Isaac"
+date: 2022-09-12
+published: true
+---
+
+[This book](https://www.amazon.com/Super-Pumped-Battle-Mike-Isaac-ebook/dp/B07PZ2B85Y/ref=tmm_kin_swatch_0?_encoding=UTF8&qid=1663623752&sr=8-1) was a total page turner. It has everything you'd want from a good story 
```

Comment:
`you'd` -> `you would`?  I'm not sure if your going for formal english, or trying to make the article have more of a conversational feel.  If you want a more conversational and relaxed feel then leave it as is.